content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -264,15 +267,23 @@ updates:
     schedule:
       interval: "weekly"
     commit-message:
-      # Prefix all commit messages with "npm"
+      # Prefix all commit messages with "npm: "
       prefix: "npm"
 
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      # Prefix all commit messages with "[docker] " (no colon, but a trailing whitespace)
+      prefix: "[docker] "
+
   - package-ecosystem: "composer"
     directory: "/"
     schedule:
       interval: "weekly"
-    # Prefix all commit messages with "Composer"
-    # include a list of updated dependencies
+    # Prefix all commit messages with "Composer" plus its scope, i. e. a list
```

Comment:
We don't use `i.e.` in the GitHub docs 
```suggestion
    # Prefix all commit messages with "Composer" plus its scope, such as a list
```

Reply:
Oh okay, I wasn't aware of that.

I'm not a native speaker, but imho "such as" sounds wrong here. I think it should be "that is", because I don't want to give an example, but I want to define what a "scope" is. I updated `such as` to `that is` now. If you have concerns about that, feel free to revert my commit :)

Reply:
I am happy with your commit 😃 👍🏻